src/diverse_seq/cli.py

```diff
@@ -449,7 +451,9 @@ def ctree(
     sketch_size: int | None,
     distance: str,
     canonical_kmers: bool,
+    limit: int | None,
     numprocs: int,
+    hide_progress: bool,
```

Comment:
making argument names consistent with cogent3 (`show_progress`) makes it easier for a user to guess what the argument name is for a given option.

Reply:
Ah, I changed from `with_progress` to `hide_progress` it to be inline with the other cli functions. I think it's best to keep this cli argument as is to be in line with the other cli functions, but I'll change the argument in the apps.

Reply:
Apps now use `show_progress`, while the cli remains with `hide_progress` as with the other cli functions